test/winchecksec-test.cpp

```diff
@@ -149,3 +149,11 @@ TEST(Winchecksec, CFG64) {
         EXPECT_FALSE(checksec.isCFG());
     }
 }
+
+TEST(Winchecksec, NoCetCompat64) {
+    auto *path = WINCHECKSEC_TEST_ASSETS "/64/pegoat-no-cetcompat.exe";
+
+    auto checksec = checksec::Checksec(path);
+
+    EXPECT_FALSE(checksec.isCetCompat());
+}
```

Comment:
N.B.: We might want to add another "goat" binary; the repo is here: https://github.com/trailofbits/pegoat

Reply:
Added testers :)

Reply:
Thanks! Could you add test stubs for them, just ensuring `EXPECT_TRUE(checksec.isCetCompat())`?

Reply:
Hmm, I just noticed these new binaries aren't in the `pegoat` repo's build. Did you build them locally?

Reply:
Thanks!